docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
     environment:
       - PYENV_VERSION=${PYENV_VERSION:-}
       - COVERSTORE_CONFIG=${COVERSTORE_CONFIG:-/openlibrary/conf/coverstore.yml}
-      - GUNICORN_OPTS=${GUNICORN_OPTS:- --reload --workers 1 --max-requests 250}
+      - GUNICORN_OPTS=${GUNICORN_OPTS:- --reload --workers 1 --max-requests 250 --limit-request-line 8190}
```

Comment:
The configuration looks correct, but this is the covers server.  Request that you revert this change, and add the `limit-request-line` flag to line 8 (the `services.web.environment` gunicorn configurations).

Reply:
You can test your changes with this URL (you'll have to restart Docker first):

http://localhost:8080/api/get_many?keys=[%22%2Fauthors%2FOL19725A%22%2C+%22%2Fauthors%2FOL47870A%22%2C+%22%2Fauthors%2FOL19512A%22%2C+%22%2Fauthors%2FOL265373A%22%2C+%22%2Fauthors%2FOL9252762A%22%2C+%22%2Fauthors%2FOL272057A%22%2C+%22%2Fauthors%2FOL115231A%22%2C+%22%2Fauthors%2FOL233050A%22%2C+%22%2Fauthors%2FOL130624A%22%2C+%22%2Fauthors%2FOL5854950A%22%2C+%22%2Fauthors%2FOL39316A%22%2C+%22%2Fauthors%2FOL646156A%22%2C+%22%2Fauthors%2FOL1816737A%22%2C+%22%2Fauthors%2FOL19158A%22%2C+%22%2Fauthors%2FOL118066A%22%2C+%22%2Fauthors%2FOL2400759A%22%2C+%22%2Fauthors%2FOL18405A%22%2C+%22%2Fauthors%2FOL3167773A%22%2C+%22%2Fauthors%2FOL116256A%22%2C+%22%2Fauthors%2FOL894149A%22%2C+%22%2Fauthors%2FOL29497A%22%2C+%22%2Fauthors%2FOL28035A%22%2C+%22%2Fauthors%2FOL391839A%22%2C+%22%2Fauthors%2FOL1272714A%22%2C+%22%2Fauthors%2FOL20188A%22%2C+%22%2Fauthors%2FOL242120A%22%2C+%22%2Fauthors%2FOL6971637A%22%2C+%22%2Fauthors%2FOL1129235A%22%2C+%22%2Fauthors%2FOL117830A%22%2C+%22%2Fauthors%2FOL719481A%22%2C+%22%2Fauthors%2FOL232501A%22%2C+%22%2Fauthors%2FOL29751A%22%2C+%22%2Fauthors%2FOL27342A%22%2C+%22%2Fauthors%2FOL28127A%22%2C+%22%2Fauthors%2FOL27349A%22%2C+%22%2Fauthors%2FOL2684942A%22%2C+%22%2Fauthors%2FOL2672053A%22%2C+%22%2Fauthors%2FOL763568A%22%2C+%22%2Fauthors%2FOL2434607A%22%2C+%22%2Fauthors%2FOL30968A%22%2C+%22%2Fauthors%2FOL113532A%22%2C+%22%2Fauthors%2FOL9252761A%22%2C+%22%2Fauthors%2FOL35183A%22%2C+%22%2Fauthors%2FOL241940A%22%2C+%22%2Fauthors%2FOL388126A%22%2C+%22%2Fauthors%2FOL22012A%22%2C+%22%2Fauthors%2FOL115504A%22%2C+%22%2Fauthors%2FOL351449A%22%2C+%22%2Fauthors%2FOL30250A%22%2C+%22%2Fauthors%2FOL2629213A%22%2C+%22%2Fauthors%2FOL19690A%22%2C+%22%2Fauthors%2FOL217730A%22%2C+%22%2Fauthors%2FOL2225458A%22%2C+%22%2Fauthors%2FOL43339A%22%2C+%22%2Fauthors%2FOL38897A%22%2C+%22%2Fauthors%2FOL2641167A%22%2C+%22%2Fauthors%2FOL528943A%22%2C+%22%2Fauthors%2FOL7023212A%22%2C+%22%2Fauthors%2FOL30224A%22%2C+%22%2Fauthors%2FOL27096A%22%2C+%22%2Fauthors%2FOL19804A%22%2C+%22%2Fauthors%2FOL870431A%22%2C+%22%2Fauthors%2FOL3290310A%22%2C+%22%2Fauthors%2FOL2629761A%22%2C+%22%2Fauthors%2FOL26681A%22%2C+%22%2Fauthors%2FOL2658871A%22%2C+%22%2Fauthors%2FOL26459A%22%2C+%22%2Fauthors%2FOL2746225A%22%2C+%22%2Fauthors%2FOL2877553A%22%2C+%22%2Fauthors%2FOL9252772A%22%2C+%22%2Fauthors%2FOL25582A%22%2C+%22%2Fauthors%2FOL27078A%22%2C+%22%2Fauthors%2FOL230834A%22%2C+%22%2Fauthors%2FOL151775A%22%2C+%22%2Fauthors%2FOL1390518A%22%2C+%22%2Fauthors%2FOL27318A%22%2C+%22%2Fauthors%2FOL32981A%22%2C+%22%2Fauthors%2FOL2677913A%22%2C+%22%2Fauthors%2FOL2623782A%22%2C+%22%2Fauthors%2FOL24527A%22%2C+%22%2Fauthors%2FOL22583A%22%2C+%22%2Fauthors%2FOL1234592A%22%2C+%22%2Fauthors%2FOL13640A%22%2C+%22%2Fauthors%2FOL20903A%22%2C+%22%2Fauthors%2FOL588695A%22%2C+%22%2Fauthors%2FOL30929A%22%2C+%22%2Fauthors%2FOL444165A%22%2C+%22%2Fauthors%2FOL19665A%22%2C+%22%2Fauthors%2FOL233499A%22%2C+%22%2Fauthors%2FOL399880A%22%2C+%22%2Fauthors%2FOL160545A%22%2C+%22%2Fauthors%2FOL709485A%22%2C+%22%2Fauthors%2FOL199956A%22%2C+%22%2Fauthors%2FOL18319A%22%2C+%22%2Fauthors%2FOL2652589A%22%2C+%22%2Fauthors%2FOL34361A%22%2C+%22%2Fauthors%2FOL27524A%22%2C+%22%2Fauthors%2FOL3620277A%22%2C+%22%2Fauthors%2FOL26069A%22%2C+%22%2Fauthors%2FOL25788A%22%2C+%22%2Fauthors%2FOL238257A%22%2C+%22%2Fauthors%2FOL23178A%22%2C+%22%2Fauthors%2FOL3245895A%22%2C+%22%2Fauthors%2FOL29394A%22%2C+%22%2Fauthors%2FOL1966702A%22%2C+%22%2Fauthors%2FOL766512A%22%2C+%22%2Fauthors%2FOL9252760A%22%2C+%22%2Fauthors%2FOL32584A%22%2C+%22%2Fauthors%2FOL4327652A%22%2C+%22%2Fauthors%2FOL60550A%22%2C+%22%2Fauthors%2FOL24762A%22%2C+%22%2Fauthors%2FOL836192A%22%2C+%22%2Fauthors%2FOL34170A%22%2C+%22%2Fauthors%2FOL21831A%22%2C+%22%2Fauthors%2FOL35145A%22%2C+%22%2Fauthors%2FOL31439A%22%2C+%22%2Fauthors%2FOL1232266A%22%2C+%22%2Fauthors%2FOL20925A%22%2C+%22%2Fauthors%2FOL25566A%22%2C+%22%2Fauthors%2FOL2746416A%22%2C+%22%2Fauthors%2FOL24457A%22%2C+%22%2Fauthors%2FOL1548599A%22%2C+%22%2Fauthors%2FOL3324313A%22%2C+%22%2Fauthors%2FOL9148252A%22%2C+%22%2Fauthors%2FOL30091A%22%2C+%22%2Fauthors%2FOL6841734A%22%2C+%22%2Fauthors%2FOL215971A%22%2C+%22%2Fauthors%2FOL220796A%22%2C+%22%2Fauthors%2FOL2630575A%22%2C+%22%2Fauthors%2FOL399682A%22%2C+%22%2Fauthors%2FOL219856A%22%2C+%22%2Fauthors%2FOL2769423A%22%2C+%22%2Fauthors%2FOL7572551A%22%2C+%22%2Fauthors%2FOL7720935A%22%2C+%22%2Fauthors%2FOL18040A%22%2C+%22%2Fauthors%2FOL2654524A%22%2C+%22%2Fauthors%2FOL424546A%22%2C+%22%2Fauthors%2FOL341986A%22%2C+%22%2Fauthors%2FOL20928A%22%2C+%22%2Fauthors%2FOL1476438A%22%2C+%22%2Fauthors%2FOL44633A%22%2C+%22%2Fauthors%2FOL33009A%22%2C+%22%2Fauthors%2FOL137220A%22%2C+%22%2Fauthors%2FOL2684610A%22%2C+%22%2Fauthors%2FOL392767A%22%2C+%22%2Fauthors%2FOL22420A%22%2C+%22%2Fauthors%2FOL7512434A%22%2C+%22%2Fauthors%2FOL26170A%22%2C+%22%2Fauthors%2FOL2657659A%22%2C+%22%2Fauthors%2FOL403331A%22%2C+%22%2Fauthors%2FOL21569A%22%2C+%22%2Fauthors%2FOL227752A%22%2C+%22%2Fauthors%2FOL30011A%22%2C+%22%2Fauthors%2FOL9256043A%22%2C+%22%2Fauthors%2FOL113959A%22%2C+%22%2Fauthors%2FOL233569A%22%2C+%22%2Fauthors%2FOL2690089A%22%2C+%22%2Fauthors%2FOL4435045A%22%2C+%22%2Fauthors%2FOL3292262A%22%2C+%22%2Fauthors%2FOL9252763A%22%2C+%22%2Fauthors%2FOL394640A%22%2C+%22%2Fauthors%2FOL221159A%22%2C+%22%2Fauthors%2FOL768231A%22%2C+%22%2Fauthors%2FOL532655A%22%2C+%22%2Fauthors%2FOL396367A%22%2C+%22%2Fauthors%2FOL114093A%22%2C+%22%2Fauthors%2FOL215841A%22%2C+%22%2Fauthors%2FOL545371A%22%2C+%22%2Fauthors%2FOL79124A%22%2C+%22%2Fauthors%2FOL32493A%22%2C+%22%2Fauthors%2FOL218453A%22]

Reply:
That will return a JSON string if your changes are correct. Otherwise, it will display an error message.

Reply:
Hey @jimchamp the limit-request-line argument dosen't seem to work. In the error message it's still showing that the limit is set to 4094, even after passing the argument!

Reply:
This works for me when the argument is passed for the web service, here: https://github.com/internetarchive/openlibrary/blob/d44e6e305e7149f48667fa829b6aa0260c5d20ff/docker-compose.yml#L3-L18

After making the change and restarting, I can see a `Recreating openlibrary_web_1` message in the console.  This is a good indicator that the change has occurred.

What OS are you using?  Maybe that's causing some issues?  I'm running Ubuntu.